src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -263,5 +297,24 @@ private async Task<IEnumerable<CodeAction>> GetRazorCodeActionsAsync(RazorCodeAc
 
             return codeActions;
         }
+
+        private static HashSet<string> GetAllSupportedCodeActionNames()
```

Comment:
Not sure I'd use  the word "supported". To me these ring true more as "available"